### DISCUSSION TITLE
[SOLVED] I updated via ReaPack and the UI is now grey? (Dark/Light/Auto button does not help)

### DISCUSSION THREAD

**Discussion by FakenameMusic** [helgoboss/helgobox#1139] [General]
Hi there,
Paid user (if that makes a difference to software) As per post title......
UPdated via ReaPack and now PLaytime 2 UI has become shades of grey (and hard to see) as per this image:
![grey interface](https://github.com/user-attachments/assets/a63d4fcf-5d39-4e63-84a3-84add515b6ce)

**Comment by helgoboss** [comment]
Can't see the screenshot

**Reply by FakenameMusic** [reply]
Sorry deleted for privacy reasons... one moment trying to edit

**Reply by FakenameMusic** [reply]
Image has been updated in the post. Thanks Benjamin.

**Comment by helgoboss** [comment]
Thanks. Press the quick theme switcher button in the title bar a few times until you find the theme you like. You probably accidentally switched it to the experimental REAPER theme.

**Reply by FakenameMusic** [reply]
Perfect! Thank you. It wasn't immediately apparent to me that "Reaper Experimental" was actually a button! I had tried switching Dark/Light/Auto but that had done nothing.
"Problem" solved. Thank you for your patience.

**Reply by helgoboss** [reply]
Glad it worked!
